homeassistant/components/media_player/browse_media.py

```diff
@@ -38,7 +42,21 @@ def async_process_play_media_url(
 
     # convert relative URL to absolute URL
     if media_content_id[0] == "/" and not allow_relative_url:
-        media_content_id = f"{get_url(hass)}{media_content_id}"
+        if (
+            for_addon
```

Comment:
It might be not just add-ons, anything behind the supervisor could be targeted, including plugins or the supervisor itself. (although unlikely for media player)

Reply:
Same naming problem I mentioned in the other comment. If we settle on a name for "generic thing managed by supervisor" I'll update both to use it.